Boost Ctrl/Boost Ctrl/Controller/News/NewsContentViewController.swift

```diff
@@ -53,6 +54,14 @@ class NewsContentViewController: UIViewController {
 		configureTableView()
 		retrieveNews()
 		tableView.separatorStyle = .singleLine
+		
+		// Show update summary if first open after update
+		if UserDefaults.standard.string(forKey: "AppVersionForUpdateSummary") == Bundle.main.releaseVersionNumberPretty {
```

Comment:
Suggestion: If you're not doing anything if it `==` you might just want to use `!=`.